debian/patches/wazo_ari_expose_voicemail_functions

```diff
@@ -504,9 +1095,178 @@ Index: asterisk-16.4.0/rest-api/api-docs/wazo.json
 +					]
 +				}
 +			]
++		},
++		{
++			"path": "/wazo/internal/voicemails/{context}/{voicemail}/{greeting}",
++			"description": "Save a voicemail's greet message",
++			"operations": [
++				{
++					"httpMethod": "PUT",
++					"summary": "Save a voicemail's greet message.",
++					"nickname": "saveVoicemailGreeting",
++					"responseClass": "void",
++					"parameters": [
++						{
++							"name": "context",
++							"description": "Voicemail context; ex.: 'default'",
++							"paramType": "path",
++							"required": true,
++							"dataType": "string",
++							"allowMultiple": false
++						},
++						{
++							"name": "voicemail",
++							"description": "Name of the voicemail",
```

Comment:
Will this be the `voicemail[id]` or `voicemail[name]` of `confd` ?

Reply:
It's going to be the voicemail[name]